pkg/controllers/externalsecret/suite_test.go

```diff
@@ -63,15 +64,28 @@ var _ = BeforeSuite(func(done Done) {
 	err = esv1alpha1.AddToScheme(scheme.Scheme)
 	Expect(err).NotTo(HaveOccurred())
 
-	err = esv1alpha1.AddToScheme(scheme.Scheme)
-	Expect(err).NotTo(HaveOccurred())
-
 	// +kubebuilder:scaffold:scheme
 
-	k8sClient, err = client.New(cfg, client.Options{Scheme: scheme.Scheme})
+	k8sManager, err := ctrl.NewManager(cfg, ctrl.Options{
+		Scheme: scheme.Scheme,
+	})
 	Expect(err).ToNot(HaveOccurred())
+
+	k8sClient = k8sManager.GetClient()
```

Comment:
```suggestion
	// do not use k8sManager.GetClient()
	// see https://github.com/kubernetes-sigs/controller-runtime/issues/343#issuecomment-469435686
	k8sClient, err = client.New(cfg, client.Options{Scheme: scheme.Scheme})
```

Reply:
thanks a lot, i remember debugging this in the past. The kubebuilder example project & book explicitly says to use the manager's client in favor of the custom client :disappointed: 

https://github.com/kubernetes-sigs/kubebuilder/blame/master/docs/book/src/cronjob-tutorial/testdata/project/controllers/suite_test.go#L115-L116

https://book.kubebuilder.io/cronjob-tutorial/writing-tests.html

Reply:
Yeah, those kubebuilder docs may need to be updated